packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -2169,6 +2184,88 @@ const TESTS_WITH_PAYPAL_AS_PAYMENT_PROVIDER: [
   ],
 ];
 
+// prettier-ignore
+const TESTS_3MONTHCOUPON: [string, any, Record<string, any>?][] = [
+  ['subscriptionFirstInvoiceDiscountEmail', new Map<string, Test | any>([
```

Comment:
Could you add some tests for the Subsequent Invoice as well please?

Reply:
done